server/app/models/profile.rb

```diff
@@ -0,0 +1,23 @@
+class Profile < ApplicationRecord
+  # encrypt password
+  has_secure_password
+  
+
+  # Can write model association here for example
+  has_many :wishes, dependent: :destroy
+  has_many :experiences, dependent: :destroy
+  # validation
+
+  #validates_presence_of :email, :password_digest, :username
+  #validates :email, uniqueness: true, on: [:create, :update]
+  #validates :username, uniqueness: true, on: [:create, :update]
+  #validate :mentor_or_mentee
+
+
+  #def mentor_or_mentee
+  #  unless mentor || mentee
+  #    errors.add(:api_user, "must either want to be a mentor or mentee or both")
+  #  end
+  #end
+
```

Comment:
The above commented-out lines: They can be deleted if they are not used.

Reply:
I've uncommented the validation